lintreview/tools/phpcs.py

```diff
@@ -48,6 +48,9 @@ def create_command(self, files):
         standard = 'PSR2'
         if self.options.get('standard'):
             standard = self.apply_base(self.options['standard'])
+        if self.options.get('ignore'):
+            ignore = self.options['ignore']
+            command.append('--ignore=' + ignore)
```

Comment:
Does this need to be shell escaped? It might be wise to append both `--ignore=` and `ignore` into the array.